barretenberg/cpp/pil/avm/avm_mini.pil

```diff
@@ -108,4 +129,10 @@ namespace avmMini(256);
     // Same for the relations related to the error activation:
     // (ib * inv - 1 + op_div_err) = 0 && op_err * (1 - inv) = 0 
     // This works in combination with op_div_err * (sel_op_div - 1) = 0;
-    // Drawback is the need to paralllelize the latter.
\ No newline at end of file
+    // Drawback is the need to paralllelize the latter.
+
+    // Inter-table Constraints
+    
+    // TODO: tag_err {clk} IS memTrace.m_tag_err {memTrace.m_clk}
```

Comment:
does this not work at the moment?

Reply:
I did not try yet. I thought I will take care of the lookup and equivalence checks in a dedicated PR once Kesha's work is completed.

Reply:
sounds good to me!